thirdparty/breakpad/mainwindow.cpp

```diff
@@ -0,0 +1,334 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2017 Nikolaos Hatzopoulos (nickhatz@csu.fullerton.edu)
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include "mainwindow.h"
+#include "ui_mainwindow.h"
+
+bool launcher(wstring program){
+
+    STARTUPINFO si;
+    PROCESS_INFORMATION pi;
+    wstring mycmd;
+
+    mycmd = program;
+
+    ZeroMemory( &si, sizeof(si) );
+    si.cb = sizeof(si);
+    ZeroMemory( &pi, sizeof(pi) );
+
+    //DWORD pid = GetCurrentProcessId();
+
+    // Open a Windows Process equivelant to fork() for Linux
+    if( !CreateProcess( NULL,   // No module name (use command line)
+            (WCHAR *)mycmd.c_str(),          // Command line
+            NULL,           // Process handle not inheritable
+            NULL,           // Thread handle not inheritable
+            FALSE,          // Set handle inheritance to FALSE
+            0,              // No creation flags
+            NULL,           // Use parent's environment block
+            NULL,           // Use parent's starting directory
+            &si,            // Pointer to STARTUPINFO structure
+            &pi )           // Pointer to PROCESS_INFORMATION structure
+        )
+        {
+            printf( "CreateProcess failed (%lu).\n", GetLastError() );
+            return true;
+        }
+
+    // Wait until child process exits.
+    //WaitForSingleObject( pi.hProcess, INFINITE );
+
+    // Close process and thread handles.
+
+    //exit(0);
+    //CloseHandle( pi.hProcess );
+    //CloseHandle( pi.hThread );
+
+
+    Q_UNUSED(program);
+    return false;
+
+}
+
+QString get_crashreporter_path(){
```

Comment:
Maybe you can just use `qApp-> applicationDirPath()` ? That would make this code portable.
See http://doc.qt.io/qt-5/qcoreapplication.html#applicationDirPath